OsmAnd/res/layout/item_external_device.xml

```diff
@@ -62,18 +62,32 @@
 
                 </LinearLayout>
 
-                <androidx.appcompat.widget.AppCompatImageView
-                    android:id="@+id/menu_icon"
-                    android:layout_width="@dimen/icon_size_double"
-                    android:layout_height="@dimen/icon_size_double"
-                    android:layout_gravity="center"
-                    android:scaleType="center"
+                <FrameLayout
```

Comment:
Do we really need FrameLayout here? It will be bad UI if children be visible simultaneously